xclim/data/fr.json

```diff
@@ -984,39 +983,39 @@
   },
   "DRY_SPELL_FREQUENCY": {
     "title": "Fréquence des périodes sèches",
-    "abstract": "Fréquence des périodes sèches de n jours et plus, pendant lesquelles les précipitations accumulées ou maximales sur une fenêtre de n jours sont inférieures à un seuil donné.",
+    "abstract": "Fréquence des périodes sèches de `N` jours et plus, pendant lesquelles les précipitations accumulées ou maximales sur une fenêtre de jours donné sont inférieures à un seuil donné.",
```

Comment:
J'avais l'impression qu'on voulait éviter les tournure de phrase `de N jours et plus` ?

Reply:
Si tu sais une meilleure manière d'écrire ça, go for it!

Reply:
d'une durée minimale donnée ?

Reply:
d'une durée minimale de jours donnée ?

Reply:
pas certaine qu'on gagne de l'information en rajoutant ''de jours'' et ça sonne moins bien à mon oreille personnellement.